internal/kubeclient/indexer.go

```diff
@@ -26,6 +28,9 @@ const (
 	StagesByArgoCDApplicationsIndexField = "applications"
 	StagesByUpstreamStagesIndexField     = "upstreamStages"
 	StagesByWarehouseIndexField          = "warehouse"
+
+	ServiceAccountsByGroupIndexField   = "group"
+	ServiceAccountsBySubjectIndexField = "subject"
```

Comment:
Tiniest nit: Should these be plural "groups" and "subjects" because I think both of these indices can end up being mulit-valued.

Reply:
Non-blocking.